src/oc_erchef/schema/sqitch.plan

```diff
@@ -1,6 +1,6 @@
 %syntax-version=1.0.0
 %project=enterprise_chef
-%uri=https://github.com/opscode/enterprise-chef-server-schema
+%uri=https://github.com/chef/enterprise-chef-server-schema
```

Comment:
This URI(https://github.com/chef/oc_bifrost) is looking like working with automate. Could you please update in all the sqitch.plan files. These changes can resolve #2803 issue